src/site/stages/build/process-cms-exports/schemas/input/paragraph-service_location.js

```diff
@@ -11,16 +11,16 @@ module.exports = {
         field_email_label: { $ref: 'GenericNestedString' },
       },
     },
-    field_facility_service_hours: {
-      type: 'array',
-      items: {
-        type: 'object',
-        properties: {
-          value: { type: 'array' },
-        },
-        maxItems: 1,
-      },
-    },
+    // field_facility_service_hours: {
+    //   type: 'array',
+    //   items: {
+    //     type: 'object',
+    //     properties: {
+    //       value: { type: 'array' },
+    //     },
+    //     maxItems: 1,
+    //   },
+    // },
```

Comment:
Why was this removed?

Reply:
Is it to bypass an error?

Reply:
failing builds with the tests data, wouldn't merge this either